bindings/java/src/main/native/jucx_common_def.h

```diff
@@ -21,4 +21,14 @@ JNIEXPORT void JNICALL JNU_ThrowException(JNIEnv *, const char *);
 
 void JNU_ThrowExceptionByStatus(JNIEnv *, ucs_status_t);
 
+#define JUCX_DEFINE_CONSTANT(_name) do { \
+    jfieldID field = env->GetStaticFieldID(cls, #_name, "J"); \
+    env->SetStaticLongField(cls, field, _name); \
+} while(0)
+
+#define JUCX_DEFINE_ENUM(_name) do { \
+    jfieldID field = env->GetStaticFieldID(cls, #_name, "I"); \
+    env->SetStaticIntField(cls, field, _name); \
```

Comment:
size of enum in not defined in C and C++ standard, do we have access to native C type here to check its size?

Reply:
https://stackoverflow.com/questions/6434105/are-default-enum-values-in-c-the-same-for-all-compilers
I don't think there would be so many thread modes, that exceeds max int value.

Reply:
this macro has common name and placed in "common" file, if you are going to add separate macro to every enum then plz rename it accordingly.

Reply:
This macro sets static int value in class `cls` to the value of enum field, that has the same `_name`

Reply:
I see, my concern is size of enum in C is not defined, usage of this macro in common case is not safe.